lib/Service.php

```diff
@@ -189,6 +189,27 @@ function write($rootElementName, $value, $contextUri = null) {
 
     }
 
+    /**
+     * Maps the given plain-old-php-object (popo) to the given rootElementName within the given namespace.
+     *
+     * The $rootElement must be specified in clark notation.
+     *
+     * @param string $rootElementName
+     * @param object $popo
+     * @param string $namespace
+     */
+    function mapPopo($rootElementName, $popo, $namespace) {
+        if (!is_object($popo)) {
```

Comment:
The first thing that popped in my mind is ... should it be based on classnames instead of objects? It seems a bit odd to me to pass an object instance that's then cloned...  Thoughts?